db/system_keyspace.cc

```diff
@@ -110,6 +110,19 @@ namespace {
             props.wait_for_sync_to_commitlog = true;
         }
     });
+
+    const auto set_use_schema_commitlog = schema_builder::register_static_configurator([](const sstring& ks_name, const sstring& cf_name, schema_static_props& props) {
```

Comment:
Let's put this into a separate commit, so that the infrastructure setup and refactoring existing code to use the new infra can be reviewed / merged separately.

Reply:
Putting use_schema_commitlog in the static props and annotating raft tables with it are two separate changes.